client/android/app/src/main/java/com/callout/MainApplication.java

```diff
@@ -38,7 +41,10 @@ public boolean getUseDeveloperSupport() {
             new RNDeviceInfo(),
             new MapsPackage(),
             new CodePush(getResources().getString(R.string.reactNativeCodePush_androidDeploymentKey), getApplicationContext(), BuildConfig.DEBUG),
-            new VectorIconsPackage()
+            new VectorIconsPackage(),
+            new RNFirebasePackage(),
+            new RNFirebaseMessagingPackage(),
```

Comment:
do we need both messaging and notifications? what is the difference?

Reply:
messaging is data only and handled by the end app, notifications are key pairs that are displayed by the FCM engine (as we discussed APN are handed internally so lets keep the approach the same)

Reply:
also notification doesnt have the functions to fetch a token from the server... weirdly